fileupload.go

```diff
@@ -47,7 +47,7 @@ type FileUpload struct {
 	Created  int64             `json:"created"`
 	ID       string            `json:"id"`
 	Filename string            `json:"filename"`
-	Links    []*FileLink       `json:"links"`
+	Links    *FileLinkList     `json:"links"`
```

Comment:
Nice catch, totally missed that one when I shipped the FileLink resource

Reply:
I can't get the credit on this one as I didn't catch this. I only came across as it was reported in a ticket.